.github/pull_request_template.md

```diff
@@ -1,21 +1,24 @@
 # Summary
 
-A concise description of the changes being introduced. If you are submitting a PR for a
-new quickstart please review the pre merge checklist section to validate this PR is ready for review.
+A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this PR is ready for review.
```

Comment:
After discussing, we should also add in somewhere that if a PR is not ready for review, it should remain in a draft until all parties are ready for review

Reply:
```suggestion
A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. If it is not ready, please mark the pull request as a draft.
```